progress.go

```diff
@@ -11,16 +11,25 @@ import (
 	"sync/atomic"
 )
 
-const defaultChunkDir = ".chunk"
+// DefaultChunkDir is the directory where Chunk keeps track of each chunk
+// downloaded of each file. It us created under the user's home directory by
+// default. It can be replaced by the environment variable CHUNK_DIR.
+const DefaultChunkDir = ".chunk"
 
 // get the chunk directory under user's home directory
 // TODO: make it configurable (maybe an envvar?)
-func getChunkDirectory() (string, error) {
-	u, err := user.Current()
-	if err != nil {
-		return "", fmt.Errorf("could not get current user: %w", err)
+func getChunkDirectory(custom string) (string, error) {
+	d := os.Getenv("CHUNK_DIR")
```

Comment:
Shouldn't this variable be named `dir` instead? As we use the `dir` variables on other functions and it represents the same meaning and directory

Reply:
Given the small scope of the function (15 lines), I'm OK with `d` — but offer no resistance to change it either lol.

To [cite](https://biancarosa.com.br/posts/go_clean_code_1/) @biancarosa:

> Their scope is tiny, the code block that its used is _tiny_. You can see right where it ends. You don’t have to scroll down too much.
>
> On a more general rule, your variables names should grow according to its _scope_. Don’t go all crazy and use `c` instead of `lineCount` for package variables.